acceptance-tests/src/test/java/me/atam/atam4jsampleapp/testsupport/Atam4jApplicationStarter.java

```diff
@@ -9,11 +9,12 @@
 public class Atam4jApplicationStarter {
 
     public static DropwizardTestSupport<ApplicationConfiguration> startApplicationWith(int initialDelayInMillis,
-                                                                                       Class testClass) {
+                                                                                       Class testClass, int periodInMillis) {
 
         ConfigOverride[] configOverrides = {
             ConfigOverride.config("testClasses", testClass.getName()),
-            ConfigOverride.config("initialDelayInMillis", String.valueOf(initialDelayInMillis))
+            ConfigOverride.config("initialDelayInMillis", String.valueOf(initialDelayInMillis)),
+            ConfigOverride.config("periodInMillis", String.valueOf(periodInMillis))
```

Comment:
Just a thought - it may have been better to introduce another constructor rather than changing the signature of the existing constructor. This would have reduced the overall impact (classes refactored) of the change. Example - the change in class `FailingTestAcceptanceTest.java`, `PassingAndFailingTestsByCategoryAcceptanceTest.java` etc wasn't really needed / wasn't in scope of this change.